test/functional/services/remote/webdriver.ts

```diff
@@ -98,6 +99,9 @@ async function attemptToCreateCommand(
           // See: https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md
           chromeOptions.push('headless', 'disable-gpu');
         }
+        if (certValidation === '0') {
+          chromeOptions.push('headless', 'disable-gpu', 'ignore-certificate-errors');
```

Comment:
Are these other options needed with `ignore-certificate-errors`?

Reply:
Good catch @tylersmalley those are options are for headless chrome :) I removed them.  Thanks!! :)